javascript/builtins/intl/NumberFormat.json

```diff
@@ -571,7 +571,7 @@
             },
             "options_useGrouping_parameter": {
               "__compat": {
-                "description": "<code>options.useGrouping</code> parameter",
+                "description": "<code>options.useGrouping</code> parameter accepts: <code>'always'</code>, <code>'auto'</code>, <code>'min2'</code> (as well as <code>true</code> and <code>false</code>)",
```

Comment:
@queengooborg I need advice on this:

`options_useGrouping_parameter`
-  has existed since the constructor was created but previously could only be values `true` and `false`. 
- Having no feature for this previously was "as expected" because the compatibility only shows things that are non-compatible - and if it is present since day 1.
- But now the [NumberFormat v3](https://github.com/tc39/proposal-intl-numberformat-v3#grouping-enum-ecma-402-367) changes added the options: always, auto and min2. These are only implemented in FF nightly.

The problem with this is/was that with this name it looks like the whole parameter option is not supported in Chrome (say) because you see this subfeature and it appears not to be supported for Chrome. Further it looks like the option can not be used in FF prior to preview. While in fact the values of true and false are supported from a long time ago.

What I have done is added a note about what options are supported by this subfeature. I am not sure this is sufficient because notes are not visible by default. So I might reasonably 
- split this into two features, one just following the constructor, the other being this one.
- something else?

What do you think?

Reply:
I think that if we should create a subfeature for the specific values (that is, `options_useGrouping_parameter.string_values`), so that the support for the parameter as a whole can be represented separately from the new values.  How does that sound?

Reply:
Yes. Done. Though I called it `options_useGrouping_parameter-string_values` since a period is not allowed as a separator.